dashboard/client/src/components/CopyableCollapse.tsx

```diff
@@ -0,0 +1,34 @@
+import { Tooltip } from "@material-ui/core";
+import { CheckOutlined, FileCopyOutlined } from "@material-ui/icons";
+import copy from "copy-to-clipboard";
+import React, { useState } from "react";
+
+const CopyableCollapse: React.FC<{
```

Comment:
seems like this could be handled by pure css instead.
```
display: block
text-overflow: ellipsis
text-wrap: nowrap
overflow: hidden
width: 50px
```

You can see an example here (see idCol css class): https://github.com/ray-project/ray/blob/d915529e9e368351134687f02c799d3ab85ed228/dashboard/client/src/pages/node/NodeRow.tsx#L80

This css rule will cut off the text after 50px but if a user selects it and copies it, it will copy the entire string.

Reply:
Make sense!